### ISSUE TITLE
apply() does not work on iterable

### ISSUE THREAD

**Issue by hillin** [iter-tools/iter-tools#474]
Per #473: `apply()` should work with iterable.
E.g.: the following code does not compile:

```typescript
const minValue = execPipe(
  [2, 7, 14],
  map(x => x * x),
  apply(Math.min)
);
```

> error TS2769: No overload matches this call.
  Overload 1 of 8, '(initial: number[], fn0: (value: number[]) => any[] | null | undefined, fn1: (value: any[] | null | undefined) => number): number', gave the following error.       
    Argument of type '(source: Wrappable<number>) => IterableIterator<number>' is not assignable to parameter of type '(value: number[]) => any[] | null | undefined'.

**Comment by conartist6** [comment]
Ah. The code does work, but you're having problems with the types. Here, I put all the relevant types in [a playground](https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBDAnmYcAyEJgIYCMA2wAPACoB8cAvHAJIzBR6GlkDcAUKJLAsqgOqMwOAsXJV0mEc3EAfOADsArvnxx5ShQBNgAMwCWC4Fo7sdAY3zYoqXZvMx9EBXAC22MEQDyAGjjkACnY4ODsFcwAuOACAN2x8JWAokj99KOVXXAYASioKX3ZsqICAZwglKHMkuEEPaTEyXMoKOgYmYFbGGGhvNnYzYEtrW3tHZzcPbz9A4ND7YriE6pS4NMUlTJy8uAKQsoqqqNrhdpYfQqjO9qvuqF7TAHoHuDAbGEcGAFp9AHMFaGAAyGNjm4TGLlAgwACvoUEQAIIUAKGfSOeJReFFODw9hPF5vD5Qb5-AFAqwgsIOJwQkDQ2HEeF+ABKAEYkbMUWj8BjziFdAoAAwLeKJDFNCis85Y1kcPGvYDvfRfX7-Gxk4agqnjSHmGFwxlwSWGgBM7JCnP06OxvNCguFSzF20ls35LPtosNLPFJqlUSZxtlz3liuVJLVFnJIzB1LgOr1DOZLOZxuZAGYzXBFh7GS67dEs9VMU6k7m3fmRdVWd7-Tb+cb3ZXjdXU77DanA-iFYTiarARGNZTwbHabr6QjE8m08yACwZi1WnN8vOxCuO5qe2sKMsrh2e6sp3P18u7-3Nzepht+1PV6etpnTjvB7sq0n9imjGNxscGo01tsz5kAFY5wUVFLW5a1cyFY9s2rEsl23As-S9J0DyXI8dw9U8nRbXMLxgytrydW9c2nS9DWnatALvQDHwJJUiRfcNBkjTUhy-fUJx9f8KKA5kADYQLAhdN2gzDCzgzdENXPdUM3DCkJNM88PIpkiPXe9NzIgi-Uop1qNzQDVMA6t+Lvfi6K7Bie1fFiBw-bUR3jFhog5UCuWSG0ADofP5EoMSgRhECIcTkm9cgLn8UwuGgeAAG9hzpFA4AAX1MdV32jcY6nwYKmQVCoFD8REgiXYofK86wfn87FAuwYLEWrAqoCK2Yqpqq5RARCh5GUVR1DgTQdAMIwtDvZqFFMN8oy1FwcoaqBquHehtBKWqgqIbAFEQMhmQmshSttcqfPagLqqamBCptU7auq8bLpaqa7My2a4Hmoh8oehQDtzY7KsWmr4Tq4Ktp2i6rsigIbqBjbQZ69Z+o0bQ9EMYxwce-p2HMZwSngVxDAANQrCQOOAQ6AG0UzgAB2PwWWnABdG13DAAIQG2dmACo4BAbJruEXKAgAWWwGAAAsvPxhRskKVggA).

I'll get you a fix shortly.

**Comment by conartist6** [comment]
Fix released in `v7.3.2`

**Comment by hillin** [comment]
Fantastic! Quickest support I've ever seen 🥇

**Comment by hillin** [comment]
Can confirm it's fixed in `v7.3.2`.

**Comment by conartist6** [comment]
Heh. Now if only I could get the rest of the OSS community to prioritize support too! We might all be a little saner.
